app/controllers/posts_controller.rb

```diff
@@ -49,6 +49,14 @@ def create
     @post = Post.new(post_params.merge(user: current_user, body: helpers.post_markdown(:post, :body_markdown),
                                        category: @category, post_type: @post_type, parent: @parent))
 
+    if @post.title?
+      if @post.title.include? "$$"
+        flash[:danger] = I18n.t 'posts.no_block_mathjax_title'
+        redirect_back fallback_location: root_path
```

Comment:
Sorry, missed this one - last one! Can we make this a `render :new, status: :bad_request` instead, so that we keep the post details submitted.

```suggestion
        render :new, status: :bad_request
```

Reply:
Did you mean if user enter's "$$" than, we let them post that?

Reply:
No, that doesn't actually post it. Using `redirect` means that the form will be cleared. Using `render` means that the text will stay so it can be edited.